sdk/sdk-api/src/main/java/mil/emp3/api/utils/ColorUtils.java

```diff
@@ -41,4 +42,26 @@ public static String colorToString(IGeoColor oColor) {
         
         return sStr;
     }
+
+    /**
+     * Returns the IGeoColor of the given hex string
+     * @param hexColorValue - Hex color in String form. Assuming the form of #RRGGBB
+     * @return - IGeoColor object representing the given string.
+     */
+    public static IGeoColor stringToColor(final String hexColorValue) {
+        final int radix = 16;
+        try {
+            int red = Integer.valueOf(hexColorValue.substring(1, 3), radix);
+            int green = Integer.valueOf(hexColorValue.substring(3, 5), radix);
+            int blue = Integer.valueOf(hexColorValue.substring(5, 7), radix);
+            final IGeoColor geoColor = new GeoColor();
+            geoColor.setRed(red);
+            geoColor.setGreen(green);
+            geoColor.setBlue(blue);
+            return geoColor;
+        }
+        catch (Exception exception) {
```

Comment:
Is there a more specific exception that you can catch here? It is better to have a specific exception

Reply:
Use EMP_Exception

Reply:
Revised to catch number format exception for integer parsing and checked the length of the string.